contrib/cmake/CMakeLists.txt

```diff
@@ -89,7 +89,7 @@ if( ${WINDOWS} )
     set( PLATFORM_LIBS winmm.lib )
 endif()
 
-set( CORTEX_SOURCE_DIR ${CMAKE_SOURCE_DIR}/../.. )
+set( CORTEX_SOURCE_DIR ${CMAKE_SOURCE_DIR}/../..)
```

Comment:
Being pedantic here but that space on the end looks better :) can ignore

Reply:
I appreciate the consistency - fixed that one and a few others as well.